core/domain/platform_parameter_domain.py

```diff
@@ -95,14 +95,14 @@ def browser_type(self):
 
     @property
     def app_version(self):
-        # TODO(#11208): Update the documentation below to reflect the change
-        # when the GAE app version is used for web & backend.
         """Returns client application version.
 
         Returns:
             str|None. The version of native application, e.g. '1.0.0',
             None if the platform type is Web.
         """
+        # TODO(#11208): Update the documentation below to reflect the change
```

Comment:
This should read 'Update the documentation above' since it was moved.

Reply:
Oh I missed that 😅. Sorry